backend/src/proto/gradido.proto.d.ts

```diff
@@ -0,0 +1,1943 @@
+            public static encode(message: proto.gradido.ITransactionBody, writer?: $protobuf.Writer): $protobuf.Writer;
+
+            /**
+             * Encodes the specified TransactionBody message, length delimited. Does not implicitly {@link proto.gradido.TransactionBody.verify|verify} messages.
+             * @param message TransactionBody message or plain object to encode
+             * @param [writer] Writer to encode to
+             * @returns Writer
+             */
+            public static encodeDelimited(message: proto.gradido.ITransactionBody, writer?: $protobuf.Writer): $protobuf.Writer;
+
+            /**
+             * Decodes a TransactionBody message from the specified reader or buffer.
+             * @param reader Reader or buffer to decode from
+             * @param [length] Message length if known beforehand
+             * @returns TransactionBody
+             * @throws {Error} If the payload is not a reader or valid buffer
+             * @throws {$protobuf.util.ProtocolError} If required fields are missing
+             */
+            public static decode(reader: ($protobuf.Reader|Uint8Array), length?: number): proto.gradido.TransactionBody;
+
+            /**
+             * Decodes a TransactionBody message from the specified reader or buffer, length delimited.
+             * @param reader Reader or buffer to decode from
+             * @returns TransactionBody
+             * @throws {Error} If the payload is not a reader or valid buffer
+             * @throws {$protobuf.util.ProtocolError} If required fields are missing
+             */
+            public static decodeDelimited(reader: ($protobuf.Reader|Uint8Array)): proto.gradido.TransactionBody;
+
+            /**
+             * Verifies a TransactionBody message.
+             * @param message Plain object to verify
+             * @returns `null` if valid, otherwise the reason why it is not
+             */
+            public static verify(message: { [k: string]: any }): (string|null);
+
+            /**
+             * Creates a TransactionBody message from a plain object. Also converts values to their respective internal types.
+             * @param object Plain object
+             * @returns TransactionBody
+             */
+            public static fromObject(object: { [k: string]: any }): proto.gradido.TransactionBody;
+
+            /**
+             * Creates a plain object from a TransactionBody message. Also converts values to other types if specified.
+             * @param message TransactionBody
+             * @param [options] Conversion options
+             * @returns Plain object
+             */
+            public static toObject(message: proto.gradido.TransactionBody, options?: $protobuf.IConversionOptions): { [k: string]: any };
+
+            /**
+             * Converts this TransactionBody to JSON.
+             * @returns JSON object
+             */
+            public toJSON(): { [k: string]: any };
+        }
+    }
+}
```

Comment:
What the hell is this file doing? Nearly 2000 lines of code what for?

Reply:
This whole file is generated by the proto library. I checked in the generated file since I do not want to require everyone to build this stuff everytime - just have it statically linked till we remove it again.